packages/bootstrap-chat/src/composed/BaseFrontend.jsx

```diff
@@ -35,10 +35,12 @@ function BaseFrontend({
   const { connectionStatus, agentStatus, taskConfig } = mephistoContext;
   const { appSettings } = appContext;
   const sidePaneSize = appSettings.isCoverPage ? "col-xs-12" : "col-xs-4";
+  const heightStyle =
+    taskConfig.frame_height == 0 ? {} : { height: taskConfig.frame_height };
```

Comment:
i generally like to use `===` triple equals. for example:

```
false == 0 // true
false === 0 // false

```

Reply:
Interesting, I hadn't considered this when I wrote the code, but think that the implicit conversions in this case are actually reasonable.